.env

```diff
@@ -0,0 +1,11 @@
+GITHUB_DEVTRIAL_APP_ID=971b69f8cf48708cf1a5
+GITHUB_DEVTRIAL_APP_SECRET=c332ee3e9830f9177d2a4cb7b3e3ba377395d4fa
+TWITTER_DEVTRIAL_APP_ID=DryR6Y1ts4f6sxr9Y0snd0D5V
+TWITTER_DEVTRIAL_APP_SECRET=j8rmlZCKXdtzLYFHD7tBtgG8N9XQ01IW0V2tcSWFIYoWY9X57Z
+LI_DEVTRIAL_APP_ID=77f3askso2dyjx
+LI_DEVTRIAL_APP_SECRET=vxSNudDSlzf9KYae
+DEVTRIAL_EMAIL_DOMAIN=gmail.com
+DEVTRIAL_EMAIL=springestdevs@gmail.com
+DEVTRIAL_EMAIL_PASSWORD=Jointh3devs
```

Comment:
Anyways, all secrets are reset, so no biggie. Thanks for pointing this out @Tuxified!